mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This is a bridge module between Management System and Hazard
+
+This module contains some new features for Management System modules.
+
+Nonconformity (NC)
+
+- Add the hazard_ids field to link specific hazards.
```

Comment:
```suggestion
This module allows the user to relate hazards to a nonconformity.
```

Reply:
Adjustments completed, I only kept the new line mentioned